docs/sources/plugins/developing/dataframe.md

```diff
@@ -0,0 +1,156 @@
++++
+title = "Introduction Dataframes"
+type = "docs"
+[menu.docs]
+name = "Dataframes"
+parent = "developing"
+weight = 9
+draft = true
++++
+
+## Introduction to dataframes
+
+Dataframes were introduced in Grafana 7.0. They replace the Time Series and Table types in Grafana. Dataframes are a more generic structure that can hold different shapes of time series, tables, and other types.
+
+A dataframe is a [Columnar oriented](https://en.wikipedia.org/wiki/Column-oriented_DBMS) table structure, meaning it stores data by column and not by row.
+
+**Simplified dataframe model:**
+
+```ts
+interface DataFrame {
```

Comment:
Have we agreed on whether it's "dataframe" or "data frame". This interface name seems to indicate that it's two words, but the rest of the article uses one word.

Reply:
It's DataFrame in the frontend. https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/types/dataFrame.ts#L108 It's Frame in the backend (avoiding studder)
https://github.com/grafana/grafana-plugin-sdk-go/blob/master/data/frame.go#L29

Googling differents combinations returns different results. 

Given the code I suggest we use two words.

Reply:
In case no one has a counter-argument, I also suggest we use two words 👍